src/algorithm/distance.rs

```diff
@@ -131,6 +131,12 @@ impl<T> Distance<T, Polygon<T>> for Point<T>
         }
         // minimum priority queue
         let mut dist_queue: BinaryHeap<Mindist<T>> = BinaryHeap::new();
+        // we've got interior rings
+        if polygon.1.len() > 0 {
```

Comment:
This outer `if` isn't necessary. If it's empty, the `for` loop won't iterate, which is fine.